runtime/include/arch/aarch64/context.h

```diff
@@ -109,6 +105,35 @@ arch_context_switch(struct arch_context *a, struct arch_context *b)
 	return 0;
 }
 
+
+/**
+ * Load a new sandbox that preempted an existing sandbox, restoring only the
+ * instruction pointer and stack pointer registers.
+ * @param active_context - the context of the current worker thread
+ * @param sandbox_context - the context that we want to restore
+ */
+static inline void
+arch_context_restore_fast(mcontext_t *active_context, struct arch_context *sandbox_context)
```

Comment:
Can you please reorder this to be above `arch_context_switch`? That would make sure the order is the same as the Intel equivalent file.